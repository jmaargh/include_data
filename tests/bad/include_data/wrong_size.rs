fn main() {}

static TOO_SHORT: [u16; 30] = include_data::include_data!("../../test_data/binary_64");
