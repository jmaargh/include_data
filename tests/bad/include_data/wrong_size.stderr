error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
 --> tests/bad/include_data/wrong_size.rs:3:31
  |
3 | static TOO_SHORT: [u16; 30] = include_data::include_data!("../../test_data/binary_64");
  |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: source type: `[u8; 64]` (512 bits)
  = note: target type: `[u16; 30]` (480 bits)
  = note: this error originates in the macro `include_data::include_data` (in Nightly builds, run with -Z macro-backtrace for more info)
